packages/dev/loaders/src/glTF/2.0/Extensions/Interactivity/interactivityFunctions.ts

```diff
@@ -0,0 +1,230 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import type { IKHRInteractivity, IKHRInteractivity_Configuration, IKHRInteractivity_Node, IKHRInteractivity_Variable } from "babylonjs-gltf2interface";
+import type { IFlowGraphBlockConfiguration } from "core/FlowGraph";
+import type { ISerializedFlowGraph, ISerializedFlowGraphBlock, ISerializedFlowGraphConnection, ISerializedFlowGraphContext } from "core/FlowGraph/typeDefinitions";
+import { RandomGUID } from "core/Misc";
+import { gltfPropertyNameToBabylonPropertyName, gltfToFlowGraphTypeMap, gltfTypeToBabylonType } from "./utils";
+
+function convertValueWithType(configObject: IKHRInteractivity_Configuration, definition: IKHRInteractivity) {
+    if (configObject.type !== undefined) {
+        // get the type on the gltf definition
+        const type = definition.types && definition.types[configObject.type];
+        if (!type) {
+            throw new Error(`Unknown type: ${configObject.type}`);

```

Comment:
When an error is thrown in the glTF loader, it should have the `context` for where the parsing is happening. Otherwise, it will be difficult for a user to know where the error occurred. This is why almost every function has a `context` string that is the JSON pointer for where the parsing is happening.

Reply:
Added context pointers on the error messages 🙂